pkg/printers/.import-restrictions

```diff
@@ -1,10 +0,0 @@
-{
-  "Rules": [
-    {
-      "SelectorRegexp": "k8s[.]io/kubernetes/pkg/(api$|apis/)",
-      "ForbiddenPrefixes": [
-        "k8s.io/kubernetes/pkg/printers"
```

Comment:
why can this go?

Reply:
There is a bug in the regular expressions in this import-restrictions file, so the forbidden package can never be selected by the `SelectorRegexp`.

If the intention was to forbid imports of `k8s.io/kubernetes/pkg/{api,apis}`, then I just updated the rule to reflect that.

Reply:
I think that was the case here: no internal package references from printers. @soltysh might know details.